build.rs

```diff
@@ -21,8 +21,18 @@ fn main() {
     #[cfg(feature = "overwrite")]
     let outdir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap()).join("src/sys");
 
-    bindgen::Builder::default()
-        .header_contents("include-file.h", INCLUDE)
+    let mut builder = bindgen::Builder::default();
+
+    if let Some(path) = env::var("BUILD_IO_URING_INCLUDE_FILE")
+        .ok()
+        .filter(|path| !path.is_empty())
+    {
+        builder = builder.header(path);
+    } else {
+        builder = builder.header_contents("include-file.h", INCLUDE);
+    }
+
+    builder
```

Comment:
When I have jury-rigged build.rs to build from a local io_uring.h in my home directory, I had to add ".clang_arg("-F.") to this builder chain. With the change to using builder.header(path) above, that's not necessary now?